cfme/cloud/object_store.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+import cfme.fixtures.pytest_selenium as sel
+from cfme.common import SummaryMixin, Taggable
+from cfme.web_ui import mixins
+from cfme.web_ui import CheckboxTable
+
+
+list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
+
+
+class ObjectStore(Taggable, SummaryMixin):
+    """ Automate Model page of Cloud Object Stores
+
+    Args:
+        name: Name of Object Store
+    """
+
+    def __init__(self, name=None):
+        self.name = name
+
+    def tag(self, tag, **kwargs):
```

Comment:
These shuld take Tag objects, not just a tuple, there is a mixin for tags, it could be worth seeing if it will support your use case.